pyvecorg/templates/index.html

```diff
@@ -237,10 +237,13 @@ <h3>{{ meta.bank_account.heading }}</h3>
                         <h3>{{ meta.downloads.heading }}</h3>
                         <p class="downloads">
                             <i class="fa fa-file-pdf-o" aria-hidden="true"></i>
-                            <a href="{{ url_for('static', filename='bylaws/stanovy.pdf') }}">{{ meta.downloads.bylaws }}.pdf</a>
+                            <a href="{{ url_for('static', filename='bylaws.pdf') }}">{{ meta.downloads.bylaws }}.pdf</a>
```

Comment:
I'm first linking a canonical PDF version of the bylaws, which is signed. It is actually _that_ document which our lawyers sent to the court, it's _that_ PDF downloaded from justice.cz, as is.

Then as an alternative to this terrible PDF, I'm linking our transcription to reStructuredText, which is actually the human-friendly place where we think people will read the bylaws.